### DISCUSSION TITLE
该如何隐藏 exchange-client 的导入 / 导出 excel 选单按钮

### DISCUSSION THREAD

**Discussion by mukiwu** [dream-num/univer#3035] [Q&A | 常见问题] [answered]
我已经找过 sheets-ui 的 API 文档，也有把 log 印出来，但就是找不到这两个选单的 ID 
想上来询问我该怎么隐藏这一整个 group 的按钮

![截图 2024-08-13 13 40 33](https://github.com/user-attachments/assets/190a0905-ff9a-415d-8e17-60ea4ec000b1)

**Comment by jikkai** [comment, marked as answer]
If you don't need this menu item, the best way is not to register the *exchange-client* plugin. If you must have the ability of this plugin but don't want a menu item, the corresponding id should be `sheets-exchange-client.operation.exchange`

如果不需要这个菜单的话最好的方法是不用 *exchange-client* 插件，如果一定要这个插件的能力但不希望有菜单栏的话，对应的 id 应该是 `sheets-exchange-client.operation.exchange`

```
univer.registerPlugin(UniverSheetsExchangeClientPlugin, {
  menu: {
    'sheets-exchange-client.operation.exchange': {
      hidden: true,
    },
  },
});
```

**Reply by mukiwu** [reply]
感谢！因为我有另外写 button 去串接 API 做导入导出，所以就想隐藏原有的按钮

**Comment by zhang-ke-ke** [comment]
你好，你的导入导出是基于他的后端服务吗？如果不是，能否讲一下实现思路

**Reply by mukiwu** [reply]
我是使用他們的後端服務

**Reply by zhang-ke-ke** [reply]
感谢！好吧，只需要部署他们的导入导出功能去部署后端服务，整个人好难受

**Reply by mukiwu** [reply]
沒使用他們的導入導出好像沒辦法做 ＱＱ，我也是請後端工程師協助部署

**Reply by zhang-ke-ke** [reply]
嗯嗯，我也不会部署，哈哈。已经在他的功能建议的帖子下提需求了，希望能被关注到，希望各位投投票

**Reply by mukiwu** [reply]
沒問題
